devDocs/conf.py

```diff
@@ -100,6 +100,11 @@ class AppArgs:
 # autodoc can only mock modules, not objects.
 from sphinx.ext.autodoc.mock import _make_subclass  # noqa: E402
 
+# #11971: NVDA is not running, therefore app dir is undefined.
+# Therefore tell NVDA that apt source directory is app dir.
+import globalVars  # noqa: E402
```

Comment:
`GlobalVars` is already imported above when setting `appArgs`. Also it makes sense to set `appdir` just below the line which sets `appArgs to group tweaks to `globalVars` in one place.